sidebarsDocs.js

```diff
@@ -303,6 +303,17 @@ const sidebarsDocs = {
                 'operating-scs/components/status-page/docs/components',
                 'operating-scs/components/status-page/docs/levels_of_consensus'
               ]
+            },
+            {
+              type: 'category',
+              label: 'Monitoring',
+              link: {
+                type: 'generated-index'
+              },
+              items: [
+                'operating-scs/components/monitoring/docs/enable-tracing',
+                'operating-scs/components/monitoring/docs/thanos-tuning'
```

Comment:
TODO: this list of items should be adjusted after https://github.com/SovereignCloudStack/k8s-observability/pull/44 is merged

Reply:
PR https://github.com/SovereignCloudStack/k8s-observability/pull/44 has been merged and the mentioned list has been adjusted accordingly